docs/api/app.md

```diff
@@ -496,6 +496,10 @@ app.on('ready', () => {
 });
 ```
 
+### `app.releaseSingleInstance()`
+
+Clears any lock states that were held by the single instance.
```

Comment:
I think it would be good to explain what this means and why you might need to use it